README.md

```diff
@@ -92,6 +92,13 @@ bind9_packages:
     - bind9
     - dnsutils
     - haveged
+
+# Logging
+bind9_logging_enable: false
```

Comment:
I dislike the name of the variable. It implies bind doesn't log in the default config.
```suggestion
bind9_named_logging: False
```

Reply:
I agree, the name of the variable could lead to error. I renamed it in [this commit](https://github.com/systemli/ansible-role-bind9/pull/5/commits/1f6c141580ddca818b1882049fd2bc448a232906).